qodana.yaml

```
@@ -0,0 +1,9 @@
+version: "1.0"
+linter: jetbrains/qodana-python-community:2024.3
+profile:
+  name: qodana.recommended
+include:
+  - name: CheckDependencyLicenses
+    paths:
+      -
+    path:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)